krate/src/main/kotlin/hu/autsoft/krate/optional/FloatDelegate.kt

```diff
@@ -1,13 +1,13 @@
 package hu.autsoft.krate.optional
 
 import hu.autsoft.krate.Krate
+import hu.autsoft.krate.base.KeyedKrateProperty
 import hu.autsoft.krate.util.edit
-import kotlin.properties.ReadWriteProperty
 import kotlin.reflect.KProperty
 
 internal class FloatDelegate(
-    private val key: String,
-) : ReadWriteProperty<Krate, Float?> {
+    override val key: String
```

Comment:
nit: I just noticed it here, but there are other places with the same issue: missing trailing comma! 😱

Reply:
They are missing in all `xDelegate`s.

Reply:
Now this is my kind of nitpicking - fixed 😃